### PR TITLE
Add tacotron demo from torch audio

### PR DESCRIPTION
**Pull request by StrongFennecs**
## Description 
Torch audio tacotron training on ljspeech.

## Model details
This pr adds tacotron2 with synched batch norm from torch audio. 

## Steps to reproduce any training
What settings did you use, batch size, lr, number of nodes, etc

## Training results
A gist of logs, a link to tensorboard etc if relevant.

## Things done
General:
- [ ] Linting (ruff, black, isort)

If adding a demo model training script:
- [x] Atomic saving - saving is done atomically to avoid corruption
- [ ] Checkpointing - model can successfully save a checkpoint and resume.
- [ ] Completed a full run

### CODE REVIEW

**Comment by StrongMichaelS** [comment]
Hi Adam, can you do a general clean up
- delete the forked monai (put in separate repo if you want to fix / apply some patches)
- delete the lora/hello world examples, or add new PR's for them
- delete the brats v0 example (you said its just an attempt to get cycling, v2 is proper run)
- move out the brats v2 example to a new fork and create a PR for that
